commit 9d02cead9538459d09d588dd7389adede83f7913
Author: Matt Flax <flatmax@flatmax.org>
Date:   Tue Apr 4 19:20:59 2017 +1000

    Audioinjector : make the octo and pi sound cards have different driver names
    
    This patch gives the audioinjector octo and pi soundcards different driver
    names. This allows both the be loaded without clashing.

diff --git a/sound/soc/bcm/audioinjector-octo-soundcard.c b/sound/soc/bcm/audioinjector-octo-soundcard.c
index dcf403ab3763..49115c8e20ce 100644
--- a/sound/soc/bcm/audioinjector-octo-soundcard.c
+++ b/sound/soc/bcm/audioinjector-octo-soundcard.c
@@ -324,7 +324,7 @@ MODULE_DEVICE_TABLE(of, audioinjector_octo_of_match);
 
 static struct platform_driver audioinjector_octo_driver = {
 	.driver	= {
-		.name			= "audioinjector-audio",
+		.name			= "audioinjector-octo",
 		.owner			= THIS_MODULE,
 		.of_match_table = audioinjector_octo_of_match,
 	},
diff --git a/sound/soc/bcm/audioinjector-pi-soundcard.c b/sound/soc/bcm/audioinjector-pi-soundcard.c
index ef54e0f07ea0..491906bbf446 100644
--- a/sound/soc/bcm/audioinjector-pi-soundcard.c
+++ b/sound/soc/bcm/audioinjector-pi-soundcard.c
@@ -177,7 +177,7 @@ MODULE_DEVICE_TABLE(of, audioinjector_pi_soundcard_of_match);
 
 static struct platform_driver audioinjector_pi_soundcard_driver = {
        .driver         = {
-		.name   = "audioinjector-audio",
+		.name   = "audioinjector-stereo",
 		.owner  = THIS_MODULE,
 		.of_match_table = audioinjector_pi_soundcard_of_match,
        },
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1155_9d02cead9538459d09d588dd7389adede83f7913.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1155_9d02cead9538459d09d588dd7389adede83f7913.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
